src/docs/integrations/slack/index.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: "Slack Integration"
+---
+
+Based off of this [post](https://forum.sentry.io/t/how-to-configure-slack-in-your-on-prem-sentry/3463)
+
+## Create a Slack App
+
+To use Sentry’s new Slack integration you’ll need to create an app in Slack bot app. Navigate to __Your Apps__ and click __Create new App__.
+
+![Create](./create.png)
+
+After naming your app and connecting your workspace, navigate to __Basic Information__. Here you’ll find your Client ID, Client Secret, and Verification token that lets your app access the Slack API.
+
+Here’s where you’ll connect your on-premise Sentry instance to your newly created Slack app.
+<dl>
+<dt><code>system.admin-email</code></dt>
+Declared in <code>config.yml</code>.
+
+<dd>
+<markdown>
+Copy your Client ID, Client Secret, and Verification Token.
+
+Set slack.legacy-app to False unless you are supporting an old bot integration (Pre June 2020)
```

Comment:
Now that I think about this, I think this setting should default to `false` on Sentry. @MeredithAnya, thoughts?

Reply:
i thought the same, but as others mentioned its to avoid breaking existing installs

Reply:
hmm if there is a risk at all of breaking existing installs, I would want to avoid that. I almost think it would be clearer to say, if you are creating a slack app after June 2020 set it to False, cause I don't think its about the bot app being old, its about the scopes that were requested during installation. Slack's "old" bot requests the bot scope which is like old old and maybe confuse people?

Reply:
Yeah, that's true. I assumed a fresh Sentry installation meant a fresh Slack bot so we could at least have this set to `False` by default in the example config of onpremise. That said the Slack bot can still (and likely?) to be older, right?

Reply:
i suppose your right but we already merged that change in the config example 😄 Maybe we remember next time were touching the config example we can default to false

ill rejig the wording to match the config example

Reply:
infact ill make another pr to add the `signing-secret` and enforce a default of false at same time over at `onpremise`

Reply:
alight pr open over there :) this should be save to merge if all are happy